tests/test_single_job.py

```diff
@@ -205,6 +207,11 @@ def add_reviewers(self, request):
 
     @pytest.fixture()
     def options_factory(self, fusion, add_tested, add_reviewers, add_part_of):
+        if fusion is Fusion.gitlab_rebase and (
+            add_tested or add_reviewers or add_part_of
+        ):
+            pytest.skip('Modifying commit message when using GitLab API rebase is unsupported')
+
```

Comment:
I think this way of formatting is quite common in the Python world. Not sure if 

```python
    @pytest.fixture()
    def options_factory(self, fusion, add_tested, add_reviewers, add_part_of):
        if fusion is Fusion.gitlab_rebase and (add_tested or add_reviewers or add_part_of):
            pytest.skip('Modifying commit message when using GitLab API rebase is unsupported')
```

is any better in terms of readability of the `if` line.

Reply:
I think my main problem with it is that these expressions/statements:

- `add_tested or add_reviewers or add_part_of`
- `pytest.skip('...')`

...are intended on the same level, making them appear to be related somehow, when they are in fact not related at all.

As for whether this is common in the Python world, I can't tell - you are more familiar with that world than me. :slightly_smiling_face: I do think your posted snippet is more readable in fact, at least in terms of "overall readability". The `if` line is also quite fine because of the parentheses.

So I would suggest to go with that approach. :+1: If you insist, I can probably live with the "sad guy"-style indentation though. :grin:

Reply:
```python
        if fusion is Fusion.gitlab_rebase and (add_tested or add_reviewers or add_part_of):
            pytest.skip('Modifying commit message when using GitLab API rebase is unsupported')
```

it is :smiling_imp: